webpack.config.test.js

```diff
@@ -33,7 +26,6 @@ module.exports = {
             {
                 test: /\.s[ac]ss$/i,
                 use: [
-                    MiniCssExtractPlugin.loader,
```

Comment:
The HtmlWebpackPlugin can self extract CSS without additional loaders.  The `css-loader` and `sass-loader` are enough.
The MiniCssExtractPlugin is not need more.